client/src/stores/user.ts

```diff
@@ -0,0 +1,31 @@
+import { defineStore } from 'pinia'
+import axios from 'axios'
+
+const baseUrl = 'http://localhost:8081/api'
+
+export const useUserStore = defineStore('user', {
+  state: () => ({
+    authUser: null as string | null
+  }),
```

Comment:
State doesnt persistent when refreshing and opening in a new tab. Pinia should have some options to help with this :)

Reply:
Added new package @vueuse/core which gives some localstorage functionality.